tests/performance/framework.go

```diff
@@ -0,0 +1,55 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 Red Hat, Inc.
+ *
+ */
+
+package performance
+
+import (
+	"flag"
+	"os"
+	"strconv"
+
+	. "github.com/onsi/ginkgo"
+)
+
+var RunPerfTests = false
+
+func init() {
+	flag.BoolVar(&RunPerfTests, "performance-test", false, "run performance tests. If false, all performance tests will be skiped.")
```

Comment:
Instead of adding this flag, can you for now just always skip `sig-performance` in `hack/functest.sh`? We can in a follow-up probably give the scale tests their own main entrypoint, but it should be good enough for now.